frontend/simple/views/Contributions.vue

```diff
@@ -0,0 +1,195 @@
+    </div>
+
+    <div class="section columns is-desktop is-multiline c-grid">
+      <section class="column">
+        <i18n tag="h2" class="title is-3">Receiving</i18n>
+
+        <ul class="c-ul">
+          <contribution
+            v-for="contribution, index in receiving"
+            :variant="isMonetary(contribution.type) ? 'editable' : undefined"
+            v-on="isMonetary(contribution.type) ? { click: showReceivingSettings } : {}"
+          >
+            <strong v-if="isMonetary(contribution.type)">{{textGivingMonetary(index)}}</strong>
+            <strong v-else>{{contribution.what}}</strong>
+            <i18n>from</i18n>
+            {{getWho(contribution.who)}}
+            <template v-if="hasWhoElse(contribution.who)">
+              <i18n>and</i18n>
+              <tooltip>
+                <button class="gi-is-unstyled gi-is-link-inherit">{{contribution.who.length - 1}}<i18n>others</i18n></button>
+                <template slot="tooltip">
+                  <p v-for="name, index in contribution.who" v-if="index > 0">{{name}}</p>
+                </template>
+              </tooltip>
+            </template>
+          </contribution>
+        </ul>
+      </section>
+
+      <section class="column">
+        <i18n tag="h2" class="title is-3">Giving</i18n>
+
+        <ul class="c-ul" v-if="givesNonMonetary || givesMonetary">
+          <contribution
+            v-for="contribution, index in giving.nonMonetary"
+            class="has-text-weight-bold"
+            variant="editable"
+            @new-value="(value) => handleEditNonMonetary(value, index)"
+          >{{contribution}}</contribution>
+
+          <contribution v-if="givesMonetary"
+            variant="editable"
+            @click="showGivingMincomeSettings"
+          >
+            <span class="has-text-weight-bold">{{currency}}{{giving.monetary}}</span> <i18n>to other's mincome</i18n>
+          </contribution>
+        </ul>
+
+        <ul class="c-ul">
+          <contribution variant="unfilled" @new-value="submitAddNonMonetary">
+            <i class="fa fa-heart c-contribution-icon" aria-hidden="true"></i>
+            <i18n>Add a non-monetary method</i18n>
+          </contribution>
+
+          <contribution v-if="!givesMonetary" variant="unfilled" @click="addMonetaryMethod">
+            <i class="fa fa-money c-contribution-icon" aria-hidden="true"></i>
+            <i18n>Add a monetary method</i18n>
+          </contribution>
+        </ul>
+      </section>
+    </div>
+  </main>
+</template>
+<style lang="scss" scoped>
+@import "../assets/sass/theme/index";
+
+.c-grid .column {
+  @include touch {
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  @include desktop {
+    &:first-child {
+      padding-left: 0;
+    }
+
+    &:last-child {
+      padding-right: 0;
+    }
+  }
+}
+
+.c-ul {
+  margin: $gi-spacer*0.75 0;
+}
+</style>
+<script>
+import { symbol } from './utils/currencies.js'
```

Comment:
If you want to "boy/girl scout rule" this one, I just opened a minor issue: https://github.com/okTurtles/group-income-simple/issues/481

You don't need to do it for this PR, only if you want. It would require doing some replacements in other files as well.

Reply:
I'll do it later :)